layouts/default.vue

```diff
@@ -626,6 +640,11 @@ export default class DefaultLayoutsClass extends Vue {
   showDrawer = false
 
   get title() {
+    if (this.isSafari && !this.appendApplePayOnce && (this.isStaging || this.issLocalHost)) {
+      // Apple Pay only works on safari
+      this.appendApplePayOnce = true;
+      this.paymentsLinks.push(this.applePayStaging)
```

Comment:
```suggestion
      this.paymentsLinks = [...this.paymentsLinks, this.applePayStaging]
```

No need for `this.appendApplePayOnce`